wpeverest/everest-forms
# Export entries in different formats(CSV,  ODS,  JSON,  XLSX).

### All Submissions:




### Changes proposed in this Pull Request:

Previously, there is only CSV format for exporting form entries.This PR introduce different formats(CSV,  ODS,  JSON,  XLSX)  for exporting entries..


### How to test the changes in this Pull Request:

1. Go to Everest Forms Entries.
2. Scroll down and go to entries table section. There is 'Export as CSV' as a 1st option in select field with 'Export' button. 
3. Select format in which you want to export entries and click on the 'Export' button.
4. Verify, files are downloaded or not. Also verify, content is correct format or not.

### Types of changes:



### Other information:



### Changelog entry

> Enhancement - Export entries in different formats(CSV,  ODS,  JSON,  XLSX).
